routing_model/src/actions.rs

```diff
@@ -71,6 +69,14 @@ impl InnerAction {
         self.extend_current_nodes(&node_states)
     }
 
+    pub fn with_enough_work_to_relocate(mut self, nodes: &[Node]) -> Self {
+        self.our_current_nodes
+            .values_mut()
+            .filter(|state| nodes.contains(&state.node))
+            .for_each(|state| state.work_units_done = state.node.0.age);
```

Comment:
We're doing work == age for now?

Reply:
No, but for a fake, it is a good enough approximation..